components/hab/src/cli.rs

```diff
@@ -941,6 +941,8 @@ fn sub_pkg_download() -> App<'static, 'static> {
     (@arg DOWNLOAD_DIRECTORY: --("download-directory") +takes_value "The path to store downloaded artifacts")
     (@arg PKG_IDENT_FILE: --file +takes_value +multiple {valid_ident_file}
         "File with newline separated package identifiers")
+    (@arg PKG_TOML_FILE: --toml +takes_value +multiple // {valid_toml_file}
```

Comment:
It might be better to just have a single --file param... try a TOML parse first, if that fails, fall back to try a plain text format parse?  That would avoid having an extra param and also retain backward compat.